documentation/modules/overview/con-configuration-points-mirrormaker.adoc

```diff
@@ -24,8 +25,8 @@ MirrorMaker 2.0 uses:
 
 You can use MirrorMaker 2.0 in _active/passive_ or _active/active_ cluster configurations.
 
-* In an _active/active_ configuration, both clusters are active and provide the same data simultaneously, which is useful if you want to make the same data available locally in different geographical locations.
-* In an _active/passive_ configuration, the data from an active cluster is replicated in a passive cluster, which remains on standby, for example, for data recovery in the event of system failure.
+active/active cluster configuration:: An active/active configuration has two active clusters replicating data bidirectionally. Applications can use either cluster. Each cluster can provide the same data. In this way,  you can make the same data available in different geographical locations.
```

Comment:
This seems to mirror the previous module. Is that intentional? In any case, the same about the offset synchronization applies as I commented abotu above.

Reply:
Same comment as previous.